src/course/syllabus/application/tribute-page/resources.md

```diff
@@ -0,0 +1,35 @@
+## Getting started
+
+- [Setting up a web page](https://oliverjam.es/blog/complete-guide-to-making-web-pages/)
+- [freeCodeCamp: Responsive web design](https://www.freecodecamp.org/learn/2022/responsive-web-design/)
+  > Note: sections of freeCodeCamp's curriculum used to be a part of our application requirements. We replaced it with our own projects. Around the same time, freeCodeCamp published a new project-based Responsive Web Design curriculum. It doesn't touch much on JavaScript and completing it all would take you a reasonable while, it will give you a fantastic foundation in building websites.
+
+## HTML
+
+- [freeCodeCamp: Basic HTML and HTML5](https://www.freecodecamp.org/learn/responsive-web-design/basic-html-and-html5/)
+- [W3 Schools: HTML Tutorial](https://www.w3schools.com/html/default.asp)
+
+## CSS
+
+- [freeCodeCamp: Basic CSS](https://www.freecodecamp.org/learn/responsive-web-design/basic-css/)
+- [freeCodeCamp: Responsive Web Design Principles](https://www.freecodecamp.org/learn/responsive-web-design/responsive-web-design-principles/)
+- [WebAIM: Alt Text](https://webaim.org/techniques/alttext/)
+- [CSS box model](https://www.w3schools.com/css/css_boxmodel.asp)
+
+## JavaScript
+
+- [Oli's intro to the DOM](https://oliverjam.es/blog/dom-intro/)
+- [MDN: Intro to DOM](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction)
+- [Date.toLocaleString()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString)
+
+## Git
+
+- [Git](https://git-scm.com/)
+- [GitHub's guide to set up Git](https://docs.github.com/en/get-started/quickstart/set-up-git)
```

Comment:
do we want people to set up command line git? maybe a bit advanced for this stage.

Reply:
Replying to all the Git process questions here:

I think you're right that it'd make sense to step the learning of it, and not introduce the command line version here. We maybe are happy with people using the GUI or a REPL integration for the first two projects, Desktop for the next two, and cmd line for the last two. In fact, I'm not sure we need them on cmd line until completing the application.

But whatever way we introduce it, it needs to be calculated with a vision across all five projects not in isolation. So perhaps we can do it as a separate step?

Reply:
Context from the current cohort: at least a few people have never used the Git CLI—some used Replit's GH integration to publish from there, some used the GitHub Desktop app. This is a bit of a problem as the start of the Skills Bootcamp definitely assumed a certain proficiency with the CLI.

Do you think it's easier for people to learn 2 or 3 different ways to commit as they progress? Or would it be less confusing to just learn the CLI from the beginning?

Reply:
Might be simpler to just use VS Code's Git tab, since it's pretty lean and focused entirely on staging/committing. GitHub Desktop does _everything_ Git can do (and more)

Reply:
Yeah, that's a good shout. I'm in favour of introducing VS Code early and not relying on online IDE's

Reply:
yeah vscode's git tab is v cool.

https://yourbrainoncomputers.com/using-git-with-visual-studio-code-the-ultimate-guide/
https://adamtheautomator.com/visual-studio-code-git/

these seem like reasonable resources to give people?

Reply:
yep sounds good!!